src/compiler/commandLineParser.ts

```diff
@@ -2099,8 +2186,16 @@ namespace ts {
 
         const literalFiles = arrayFrom(literalFileMap.values());
         const wildcardFiles = arrayFrom(wildcardFileMap.values());
+        const projectReferences = spec.referencesSpecs && spec.referencesSpecs.map((r): ProjectReference => {

```

Comment:
Why do you need this here? This would get updated only when there is update to tsconfig.json. So you shouldnt want this in here?

Reply:
I don't understand the comment - can you expand on that?